Do not print "e: " prefix for matches when -e is used

We call a rule group `e` for `-e` mode; we should probably avoid printing it as a prefix.

```
$ cat ~/CODE/go/hello.go
package main

func main() {
	var x map[string]int
	x["x"] = 54
	var y []int
	y[0] = 5
}

$ ruleguard -e 'm.Match("$x[$_] = $_").Where(m["x"].Type.Is("map[$_]$_"))' ~/CODE/go/hello.go
/home/isharipov/CODE/go/hello.go:5:2: e: x["x"] = 54
                                      ^ this
```

--- a/_test/install/gitclone/expected5.txt
+++ b/_test/install/gitclone/expected5.txt
@@ -1,2 +1,2 @@
-/root/target.go:10:10: e: add((1), 2)
-/root/target.go:11:10: e: add(1, (2))
+/root/target.go:10:10: add((1), 2)
+/root/target.go:11:10: add(1, (2))

--- a/_test/install/gitclone_nobuildtag/expected2.txt
+++ b/_test/install/gitclone_nobuildtag/expected2.txt
@@ -1,2 +1,2 @@
-/root/src/target.go:10:10: e: add((1), 2)
-/root/src/target.go:11:10: e: add(1, (2))
+/root/src/target.go:10:10: add((1), 2)
+/root/src/target.go:11:10: add(1, (2))

--- a/_test/install/gitclone_toolsgo/expected2.txt
+++ b/_test/install/gitclone_toolsgo/expected2.txt
@@ -1,2 +1,2 @@
-/root/src/target.go:10:10: e: add((1), 2)
-/root/src/target.go:11:10: e: add(1, (2))
+/root/src/target.go:10:10: add((1), 2)
+/root/src/target.go:11:10: add(1, (2))

--- a/analyzer/analyzer.go
+++ b/analyzer/analyzer.go
@@ -96,9 +96,10 @@ func runAnalyzer(pass *analysis.Pass) (interface{}, error) {
 		Sizes:        pass.TypesSizes,
 		Fset:         pass.Fset,
 		Report: func(info ruleguard.GoRuleInfo, n ast.Node, msg string, s *ruleguard.Suggestion) {
-			fullMessage := info.Group + ": " + msg
+			fullMessage := msg
 			if printRuleLocation {
-				fullMessage += fmt.Sprintf(" (%s:%d)", filepath.Base(info.Filename), info.Line)
+				fullMessage = fmt.Sprintf("%s: %s (%s:%d)",
+					info.Group, msg, filepath.Base(info.Filename), info.Line)
 			}
 			diag := analysis.Diagnostic{
 				Pos:     n.Pos(),

